src/validators/city-validator.js

```diff
@@ -0,0 +1,31 @@
+const  {check, param}  =  require ( 'express-validator' );
+const {validateResult} = require('../helpers/response-result');
+
+const validationStateId  = [
+    param('stateID')
+        .exists().withMessage('El campo debe existir')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        .exists().withMessage("El campo debe existir")
```